### PR TITLE
Disable zscore filter if (Sequel || <=100 || nan)

### PR DESCRIPTION
**Pull request by nlhepler**

### CODE REVIEW

File: src/Evaluator.cpp [hunk 1]
```diff
@@ -33,7 +33,8 @@ Evaluator::Evaluator(std::unique_ptr<AbstractTemplate>&& tpl, const MappedRead&
         const double zScore = impl_->ZScore();
         // TODO(lhepler): re-enable this check when the zscore bits are working again
         // assert(std::isfinite(zScore));
-        if (!std::isnan(minZScore) && (!std::isfinite(zScore) || zScore < minZScore))
+        if (mr.Model != "S/P1-C1" && minZScore > -100.0 && !std::isnan(minZScore) &&
```
Comment (dalexander) [inline src/Evaluator.cpp]: string comparison in C++ using == and != is not a good idea

Comment (dalexander) [inline src/Evaluator.cpp]: This if statement is unreadable.  Please break it up into named subconditions.

Comment (nlhepler) [inline src/Evaluator.cpp]: For your first point: http://www.cplusplus.com/reference/string/string/operators/. Fully supported, see (1)--no listed caveats, uses string::compare, is obviously clear what the intent it.

For your second point, very well. Give me a minute.

Comment (dalexander) [inline src/Evaluator.cpp]: My apologies on (1), I was thinking of C strings.

**Pull request merged**